### ISSUE TITLE
If not header, first paragraph must be selected from above and to its left

### ISSUE THREAD

**Issue by 4of92000** [aCluelessDanny/typora-theme-ursine#13]
When I'm using the Ursine theme (either Polar or Umbra) and the first paragraph is not a header, trying to select it fails. Instead, its selection box (or whatever it's called) is immediately above the text and to its left, so if I click *there*, it will select the text.

Again, this only occurs on the first paragraph, and it does not occur if the text is a header. I've tried this with other themes, so I know it's an Ursine issue.

**Comment by aCluelessDanny** [comment]
Huh, that's interesting. Turns out that little hacky thing I did for the splashscreen _does_ affect that paragraph line (I essentially messed with a couple of properties of the first "line" of text, as well as a couple of other key elements). I got an idea to work around that, but it'll probably have to wait a bit since I'm stuck on finals right now.

I'll reply here once more when I put a release up with that fix, as well as a couple of other details
